frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/k8s-resource.ts

```diff
@@ -298,4 +390,9 @@ export const k8sList = (
  * See more {@link https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.requestinit.html}
  * @return A promise that resolves to the response
  * * */
-export const k8sListResource = adapterFunc(k8sList, ['model', 'queryParams', 'raw', 'requestInit']);
+export const k8sListResource: K8sListResource = adapterFunc(k8sList, [
+  'model',
+  'queryParams',
+  'raw',
```

Comment:
Still a little odd that you're not specifying `raw` in the JSDoc

Reply:
yeah, this is not being used across so in initial PR the thought was not to expose it in a new signature.

https://github.com/openshift/console/pull/10243#discussion_r732973059

![image](https://user-images.githubusercontent.com/5129024/148330013-f27c0822-92b7-4ffc-97e0-5946dd4f59f3.png)

Reply:
My only concern is that this is something that is almost self documented in a very limited way due to the signature of the `adapterFunc` and thus looks like a gap in the JSDoc. But given Sam believes there is only 1 impacted location, I suppose we could look to move away from it... not sure how we deprecate a single param though.